src/ApiCheck/BreakingChangeHandlers.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ApiCheck
+{
+    public class BreakingChangeHandlers
+    {
+        public static bool FindTypeUsingFullName(BreakingChangeContext ctx)
```

Comment:
Fair enough, I will take it out and when I don't find a type I will just spill the guts for it and all of its members, if we have a lot of noise we can bring it back